### DISCUSSION TITLE
LDAP server using plain text password

### DISCUSSION THREAD

**Discussion by FabienZa** [dexidp/dex#2573] [Q&A]
Hello,

The documentation explain that : "Dex attempts to bind with the backing LDAP server using the end user’s plain text password. Though some LDAP implementations allow passing hashed passwords, dex doesn’t support hashing [...].  
-> https://dexidp.io/docs/connectors/ldap/#security-considerations 

Why is not possible for Dex to use hashed password ? For security reasons, administrator should not read user's plain text password. It's seem to be a big limitation for using Dex.

I guess there are something blocking the possibilites for dex to use hashed password. I'm curious to know what is it and how may be I can help to fix that it if possible.

Thanks

**Comment by LisaHLi** [comment]
Any updates for the question? I have same security concern here @nabokihms
